moped-etl/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# ATD Moped Processsor
+# An ETL Processor that uses Python 3.8
+#
+FROM python:3.8-alpine3.10
+
+RUN mkdir /app && mkdir /app/tmp && mkdir /data
+
+RUN apk add bash build-base python3-dev vim
+
+RUN pip install requests awscli web-pdb sodapy boto3 knackpy
```

Comment:
@frankhereford would you move the package requirements to `requirements.txt` and pin their versions to a major release? [here](https://github.com/cityofaustin/atd-knack-services/blob/production/Dockerfile) is an example dockerfile.